src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx

```diff
@@ -1,43 +1,100 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import classNames from 'classnames';
+import find from 'lodash/find';
 
-import StrainSelector, {
-  Strain
-} from 'src/content/app/species-selector/components/strain-selector/StrainSelector';
+import {
+  handleSelectedSpecies,
+  clearSelectedSearchResult,
+  deleteSpeciesAndSave
+} from 'src/content/app/species-selector/state/speciesSelectorActions';
+import {
+  getCurrentSpeciesGenomeId,
+  getCommittedSpecies
+} from 'src/content/app/species-selector/state/speciesSelectorSelectors';
+
+import InlineSVG from 'src/shared/inline-svg/InlineSvg';
+
+import {
+  CommittedItem,
+  PopularSpecies
+} from 'src/content/app/species-selector/types/species-search';
 
 import styles from './PopularSpeciesButton.scss';
 
+import { RootState } from 'src/store';
+
+type OwnProps = {
+  species: PopularSpecies;
+};
+
 type Props = {
+  species: PopularSpecies;
   isSelected: boolean;
-  species: string;
-  strains: Strain[];
-  onClick: () => void;
-  onStrainSelect: () => void;
+  isCommitted: boolean;
+  handleSelectedSpecies: (species: PopularSpecies) => void;
+  clearSelectedSpecies: () => void;
+  deleteCommittedSpecies: (genome_id: string) => void;
+};
+
+// FIXME: this should be moved to a file with general functions
+const ensureHost = (url: string) => {
+  return /^https?/.test(url)
+    ? url
+    : `${location.protocol}//${location.host}${url}`;
 };
```

Comment:
This is what I'd like your opinion about. We currently have `api-service`, which used to send requests only to the api backend provided. This is unusable e.g. in development, when sample svgs are served by the running webpack dev server. What I am currently doing is I updated the api service so that it can take an optional `host` parameter to use it instead of the default host of the backend, and it also checks whether the provided url is absolute or relative (if it's absolute, then the api-service won't concatenate it with the api host url).

It works, but I am still not sure as to whether this is a good interface for the service.

Reply:
> What I am currently doing is I updated the api service so that it can take an optional host parameter to use it instead of the default host of the backend

Having the `host` as an optional parameter sounds like a a better solution to this and I couldn't think of any particular reason why the `api-service` should always point to a single `host`.

Reply:
I guess I'll leave the updates to the api service as they are then?